demo/demo-springmvc/springmvc-client/src/main/java/org/apache/servicecomb/demo/springmvc/client/SpringMVCSchema.java

```diff
@@ -43,4 +43,16 @@ public class SpringMVCSchema {
   public String testApiExample(@RequestBody String name, HttpServletRequest request) {
     return null;
   }
+
+  @ApiResponses({
+      @ApiResponse(code = 200, response = String.class, message = "success",
+          examples = @Example({
+              @ExampleProperty(value = "wget http://localhost/springMvcSchema/testDefaultGetApiExample", mediaType = "text"),
+              @ExampleProperty(value = "{name:hello}", mediaType = "application/json"),
+              @ExampleProperty(value = "{name:hello}", mediaType = "json")})
+      )})
+  @RequestMapping(path = "/testDefaultGetApiExample")
+  public String testDefaultGetApiExample(@RequestBody String name, HttpServletRequest request) {
+    return null;
+  }
```

Comment:
Not only schema generation. Should create a server in springmvc-server and a test case in springmvc-client invoke the method.

Reply:
ok